<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SC2Switcher</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>511</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="toggleStartButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clearSettings">
          <property name="text">
           <string>Clear Settings On Next Restart</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>If SC2 is running on a different PC:  (optional)</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>SC2 PC IP Address:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipAddr"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;On your SC2 PC, open the Battle.net launcher, select SC2, click Options, Game Settings, and under SC2, check 'Additional Command Line Arguments', and paste this into the textbox:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>-clientapi 6119</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="downloadURL">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patchNotesLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Usernames</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Usernames:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="userNames">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Recent Usernames:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="recentUsernames"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>301</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Add Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameLine"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="addUsernameButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeUsernameButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Scene Switcher</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QCheckBox" name="switcherEnabled">
        <property name="text">
         <string>Scene Switcher Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>In Game:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="inGameScene"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Out of Game:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="outGameScene"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Replay:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="replayScene"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Observing:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="obsScene"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Extra Scenes</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>251</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Score Screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="scoreScreenScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="homeScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Co-op Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="coopScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Custom Games Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="customScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Replays Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="replaysScene"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>241</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="profileScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Lobby</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lobbyScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Campaign Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="campaignScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Versus Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="versusScene"/>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Collection Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="collectionScene"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Score Tracker</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>261</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Text Template:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QTextEdit" name="textTemplate"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;${tw}&lt;/span&gt; Terran Win&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;${tl}&lt;/span&gt; Terran Loss&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;${zw}&lt;/span&gt; Zerg Win&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;${zl}&lt;/span&gt; Zerg Loss&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;${pw}&lt;/span&gt; Protoss Win&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;${pl}&lt;/span&gt; Protoss Loss&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>191</width>
       <height>116</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QCheckBox" name="scoresEnabled">
        <property name="text">
         <string>Score Tracker Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="popupsEnabled">
        <property name="text">
         <string>Popups Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Name of text source:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textSourceName"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="scoresLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>200</y>
       <width>141</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>321</width>
       <height>175</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>35</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Terran</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Zerg</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Protoss</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Win&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="tWinPlus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tWinMinus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="zWinPlus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zWinMinus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="pWinPlus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pWinMinus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Loss&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="tLossPlus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tLossMinus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="zLossPlus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zLossMinus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="pLossPlus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pLossMinus">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>55</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Game Webhook</string>
    </attribute>
    <widget class="QCheckBox" name="webhookEnabled">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Webhook Enabled</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLineEdit" name="webhookEnterGame"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>431</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>URLs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="webhookURLList"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>271</width>
       <height>36</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QPushButton" name="addURLButton">
        <property name="text">
         <string>Add URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeURLButton">
        <property name="text">
         <string>Remove URL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
